### PR TITLE
Added check for template file

### LINKED ISSUE #194: Throw user-friendly error for "create-github-app" when template not found
## Description

We should throw a user-friendly error for "create-github-app" when run outside of the Git repo, when the template file we need is not found.

Today it's an unhandled execution. This should be a very quick fix for anyone to take on.

Here's the command file: https://github.com/openfaas-incubator/ofc-bootstrap/blob/master/cmd/create_github.go

### PR DESCRIPTION
**Pull request by kadern0**
## Description
Added check to verify template file exists, if not, an informative message is printed.

Fixes #194 

## How Has This Been Tested?
Run the test from project's root directory:
```
./bin/ofc-bootstrap create-github-app --root-domain test
Name: OFC elegant leavitt5	Root domain: test	Scheme: https
Launching browser: http://127.0.0.1:30010
Starting local HTTP server on port 30010
Please complete the workflow in the browser to create your GitHub App.
^C
```
Run the test from outside:
```
kaderno@kaderno-XPS-13-9360:~/go/src/github.com$ ./ofc-bootstrap/bin/ofc-bootstrap create-github-app --root-domain test
Error: Cannot find the template file. You need to run this command from the root directory of the repository.
```
## Checklist:

I have:

- [X] checked my changes follow the style of the existing code / OpenFaaS repos
- [ ] updated the documentation and/or roadmap in README.md
- [X] read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [X] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

File: cmd/create_github.go [hunk 1]
```diff
@@ -64,6 +65,11 @@ func createGitHubAppE(command *cobra.Command, _ []string) error {
 		"GitHubEvent": fmt.Sprintf("%s://system.%s/github-event", scheme, rootDomain),
 	}
 
+	_, err := os.Stat(path.Join("./pkg/github", "index.html"))
```
Comment (alexellis) [inline cmd/create_github.go]: Please can you inline this? if _, err :=

**Comment by alexellis** [comment]
Nice work 😁

File: cmd/create_github.go [hunk 2]
```diff
@@ -64,6 +65,10 @@ func createGitHubAppE(command *cobra.Command, _ []string) error {
 		"GitHubEvent": fmt.Sprintf("%s://system.%s/github-event", scheme, rootDomain),
 	}
 
+	if _, err := os.Stat(path.Join("./pkg/github", "index.html")); err != nil {
+		return fmt.Errorf("Cannot find the template file. You need to run this command from the root directory of the repository.\n ")
```
Comment (alexellis) [inline cmd/create_github.go]: Just so that you know, errors in Go cannot start with an uppercase letter or end with a `.`

Comment (alexellis) [reply cmd/create_github.go]: Can I suggest this alternative? -> `cannot find template "index.html", run this command from the ofc-bootstrap repository`

File: cmd/create_github.go [hunk 3]
```diff
@@ -64,6 +65,10 @@ func createGitHubAppE(command *cobra.Command, _ []string) error {
 		"GitHubEvent": fmt.Sprintf("%s://system.%s/github-event", scheme, rootDomain),
 	}
 
+	if _, err := os.Stat(path.Join("./pkg/github", "index.html")); err != nil {
```
Comment (alexellis) [inline cmd/create_github.go]: Spot on 👍

**Comment by alexellis** [comment]
You need to build locally before pushing. This is causing a syntax error. Please remove the newline too.

```

`some "string" goes here`
```

**Comment by kadern0** [comment]
That happens when you copy and paste without looking :( my bad!

**Pull request merged**